src/e2e/features/4-external-systems.feature

```diff
@@ -4,9 +4,12 @@
 # 
 Feature: External Systems
 
-  Scenario: Employee wants to create a new document for a zaak
+  Scenario: Employee wants to create a new document for a zaak Using the SmartDocumenbts wizard
     Given "Bob" is logged in to zac
-    And Employee "Bob" is on the newly created zaak with status "Intake"
+    And "Bob" wants to create a new zaak
```

Comment:
I think Bob wants to create a new document, not a new zaak? Or am I confused? ;-)

Reply:
Oh dear, ur ferry right yes!!!

Reply:
Oh, no; Bob is ferry correct here actually.

This Feature was depending on a previous Feature where a new zaak was created. And I think it is better to have standalone features. So for this document creation I also create a new zaak first. 

So now I am curious what ur opinion is on standalone features @edgarvonk

Reply:
That is fine. But I don't understand the flow then. I read:

```
And "Bob" wants to create a new zaak
When Employee "Bob" clicks on Create Document for zaak
[..]
```

Where then is the step where Bob created this new zaak? Which zaak is the zaak in the second line?

Reply:
Would next line be more understanble:

```Employee "Bob" clicks on Create Document for the new zaak```

Reply:
Ok yes, but my point is more: where is the new zaak created? I would expect something like this but maybe I misunderstand the idea here.. ;-)

```
And "Bob" wants to create a new zaak
When Employee "Bob" creates a new zaak
And Employee "Bob" clicks on Create Document for zaak
```

Reply:
I do understand you a bit; to me it is confusing that Bob wants something, and then suddenly there is a zaak :-/

```And "Bob" wants to create a new zaak```

Since that line actually triggers the creation; so I think from your second line of code we have problems understanding that same naming logic.

I do want to dive a bit deeper into the cucumber thing; just doing stuff based on logic there currently is.

Reply:
Closing the conversation now

Reply:
ok, clear! I don't know that much about Cucumber, so maybe that's just it. :-)